src/compas_occ/geometry/curves/nurbs.py

```diff
@@ -582,15 +583,36 @@ def closest_point(self, point: Point, return_parameter: bool = False) -> Union[P
         Returns
         -------
         Point or tuple
-            The nearest point on the curve, if ``parameter`` is false.
-            The nearest as (point, parameter) tuple, if ``parameter`` is true.
+            The nearest point on the curve, if ``return_parameter`` is false.
+            The nearest as (point, parameter) tuple, if ``return_parameter`` is true.
         """
         projector = GeomAPI_ProjectPointOnCurve(point.to_occ(), self.occ_curve)
         point = Point.from_occ(projector.NearestPoint())
         if not return_parameter:
             return point
         return point, projector.LowerDistanceParameter()
 
+    def curve_closest_parameters(self, curve: NurbsCurve, return_distance: bool = False) -> Union[Tuple[float, float], Tuple[Tuple[float, float], float]]:
```

Comment:
i would change the name to `closest_parameters_curve`. this seems more in line with the rest of `compas`.
also, would it make sense to add a variation that returns the points? `closest_points_curve`...

Reply:
I renamed it.
About the closest_points_curve I was wondering, is it okay to compute the points with the `point_at` method or should I try to get them immediately out of occ? because for the occ I don't really get their method for that because they want input points....

Reply:
ok will rename and add the other one